pkg/devcontainer/setup/setup.go

```diff
@@ -22,18 +24,17 @@ const (
 	ResultLocation = "/var/run/devpod/result.json"
 )
 
-func SetupContainer(setupInfo *config.Result, workspaceInfo *provider2.AgentWorkspaceInfo, chownWorkspace bool, log log.Logger) error {
-	// write result to ResultLocation
-	WriteResult(setupInfo, log)
-
-	// chown user dir
-	if chownWorkspace {
-		err := ChownWorkspace(setupInfo, log)
+func SetupContainer(setupInfo *config.Result, workspaceInfo *provider2.AgentWorkspaceInfo, log log.Logger) error {
+	if runtime.GOOS == "linux" || workspaceInfo.Agent.Driver == provider2.KubernetesDriver {
```

Comment:
I don't think this is correct now, the problem is that runtime.GOOS will now always be true, since this is executed within the container